README.md

```diff
@@ -0,0 +1,24 @@
+Currency converter
+=======================
+Converts between ISO 4217 currency codes. Takes 2 or 3 arguments. The last arg is opitonal. 
+
+Usage: go run main.go usd eur 100
```

Comment:
This command does not work.

`go build && ./currency-converter usd czk 100` works for building+running the code.